tests/functional/recovery_key.js

```diff
@@ -50,6 +56,11 @@ registerSuite('Recovery key', {
 
       .then(openPage(SETTINGS_URL, selectors.SETTINGS.HEADER))
       .then(testElementExists(selectors.SETTINGS.HEADER))
+
+      // Perform a slight sleep here to ensure that the checks
+      // to enable the panel have been performed.
+      .sleep(500)
+
       .then(visibleByQSA(selectors.RECOVERY_KEY.MENU_BUTTON))
```

Comment:
`click` already does a [visibleByQSA](https://github.com/mozilla/fxa-content-server/blob/e49656134afa7c437c93220646b23466cc9ccb62/tests/functional/lib/helpers.js#L183) so this isn't really needed and it may actually mean that your sleep isn't needed because the [findByCssSelector](https://github.com/mozilla/fxa-content-server/blob/e49656134afa7c437c93220646b23466cc9ccb62/tests/functional/lib/helpers.js#L180) will delay for up to 20 seconds.